lib/validators/openApiDiff.js

```diff
@@ -56,8 +56,8 @@ class OpenApiDiff {
     log.silly(`compare is being called`);
 
     let self = this;
-    var promise1 = self.processViaAutoRest(oldSwagger, 'old');
-    var promise2 = self.processViaAutoRest(newSwagger, 'new');
+    var promise1 = self.processViaAutoRest(oldSwagger, 'old', self.options.oldTagName);
+    var promise2 = self.processViaAutoRest(newSwagger, 'new', self.options.newTagName);
```

Comment:
>self.options.newTagName); [](start = 62, length = 25)

it doesn't really make sense to have the tag name in the constructor right ? that would mean if we call compare multiple times  there will be static tags. Maybe add them as optional parameters to compare

Reply:
This makes sense,  we send in two swaggers and a tag is associated with a swagger.  This seems like an ok implementation.